bin/internal/release-candidate-branch.version

```diff
@@ -0,0 +1 @@
+flutter-3.22-candidate.0
```

Comment:
This does not look right for this change, can you remove it?

Reply:
OK